test-scripts/launchTestUtilities.ts

```diff
@@ -76,6 +76,33 @@ export async function installVSCodeExtension(vsCodeExecutablePath: string, exten
     await invokeVSCodeCli(vsCodeExecutablePath, ['--install-extension', extensionIdentifier])
 }
 
+/**
+ * Alternative to the `--disable-extensions` CLI flag which allows us to
+ * selectively keep some extensions enabled.
+ *
+ * @param vsCodeExecutablePath Path to vscode CLI program.
+ * @param exceptIds List of extension ids that should *not* be disabled.
+ *
+ * @returns List of args which the caller is expected to pass to vscode CLI, of the form:
+ * ["--disable-extension", "foo.bar.baz", "--disable-extension", ...]
+ */
+export async function getCliArgsToDisableExtensions(
+    vsCodeExecutablePath: string,
+    params: { except: string[] }
+): Promise<string[]> {
+    console.log(`Disabling all VS Code extensions *except*: ${params.except}`)
+    const output = await invokeVSCodeCli(vsCodeExecutablePath, ['--list-extensions'])
```

Comment:
What is the likelihood and impact of the command's output changing?

Reply:
- Unlikely. 
- Can't think of a reasonable way to guard against a change, since the contents are dependent on whatever is installed on the machine. Could add some heuristic, but that is not worth the effort because...
- ...Impact in the worst case is that the function returns nonsense extension ids, so the `--disable-extension {nonsense}` will do nothing. 
  - Developer can inspect the log to see exactly what was passed to runTests() (this is always printed to the logs after this PR)